packages/web/src/js/Offcanvas.ts

```diff
@@ -0,0 +1,134 @@
+import BaseComponent from './BaseComponent';
+import EventHandler from './dom/EventHandler';
+import { enableToggleTrigger, enableDismissTrigger } from './utils/ComponentFunctions';
+
+const NAME = 'offcanvas';
+const DATA_KEY = 'offcanvas';
+const EVENT_KEY = `.${DATA_KEY}`;
+
+const EVENT_SHOW = `show${EVENT_KEY}`;
+const EVENT_SHOWN = `shown${EVENT_KEY}`;
+const EVENT_HIDE = `hide${EVENT_KEY}`;
+const EVENT_HIDDEN = `hidden${EVENT_KEY}`;
+
+const OFFCANVAS_BREAKPOINT = 1280;
```

Comment:
Wouldn't be useful create map, or more precisely `enum`, In the [shared/index.ts](packages/web-react/src/types/shared/index.ts)?
Or share the breakpoint values between styles and scripts via CSS variables?

Reply:
Indeed, it would! The answer probably is "export design tokens into JavaScript". And that's a little bit beyond the scope of this PR 🙂.

Reply:
Agree. Let's discuss about it on the Great Topics. 🥌

Reply:
Do not overflow VT pls, we know about that, but it is definitely not on our radar now.